lib/matplotlib/collections.py

```diff
@@ -1254,6 +1255,285 @@ def set_verts_and_codes(self, verts, codes):
+            intersection.  The t-sections containing the intersection are
+            simply clipped.
+
+            Setting *interpolate* to *True* will calculate the actual
+            intersection point and extend the filled region up to this point.
+
+        step : {{'pre', 'post', 'mid'}}, optional
+            Define *step* if the filling should be a step function,
+            i.e. constant in between *t*.  The value determines where the
+            step will occur:
+
+            - 'pre': The f value is continued constantly to the left from
+              every *t* position, i.e. the interval ``(t[i-1], t[i]]`` has the
+              value ``f[i]``.
+            - 'post': The y value is continued constantly to the right from
+              every *x* position, i.e. the interval ``[t[i], t[i+1])`` has the
+              value ``f[i]``.
+            - 'mid': Steps occur half-way between the *t* positions.
+
+        **kwargs
+            Forwarded to `.PolyCollection`.
+
+        See Also
+        --------
+        .PolyCollection.set_verts, .Line2D.set_data
+        """
+        kwargs = self._normalise_kwargs(self.axes, **kwargs)
+        polys, _ = self._make_verts_and_pts(
+            t, f1, f2, where, interpolate, step, self.axes, **kwargs)
+        self.set_verts(polys)
+
+    def _make_verts_and_pts(self, t, f1, f2, where, interpolate, step, axes, **kwargs):
+        """
+        Make verts that can be forwarded to `.PolyCollection`,
+        and pts that can be forwarded to `self._update_axes_datalim.
+        """
+        dirs = (self.t_direction, self._f_direction, self._f_direction)
+        # Handle united data, such as dates
+        if axes:
+            t, f1, f2 = axes._process_unit_info(
+                [*zip(dirs, [t, f1, f2])], kwargs)
+        t, f1, f2 = map(np.ma.masked_invalid, [t, f1, f2])
+
+        names = (d + s for d, s in zip(dirs, ("", "1", "2")))
+        for name, array in zip(names, [t, f1, f2]):
+            if array.ndim > 1:
+                raise ValueError(f"{name!r} is not 1-dimensional")
+
+        if where is None:
+            where = True
+        else:
+            where = np.asarray(where, dtype=bool)
+            if where.size != t.size:
+                raise ValueError(f"where size ({where.size}) does not match "
+                                 f"{self.t_direction} size ({t.size})")
+        where = where & ~functools.reduce(
+            np.logical_or, map(np.ma.getmaskarray, [t, f1, f2]))
+
+        t, f1, f2 = np.broadcast_arrays(
+            np.atleast_1d(t), f1, f2, subok=True)
+
+        polys = [
+            self._make_pts(t, f1, f2, idx0, idx1, step, interpolate)
+            for idx0, idx1 in cbook.contiguous_regions(where)
+        ]
+
+        pts = self._normalise_pts(np.vstack([
+            np.hstack([t[where, None], f[where, None]]) for f in (f1, f2)]))
+
+        return polys, pts
+
+    @staticmethod
+    def _normalise_kwargs(axes, **kwargs):
```

Comment:
Matplotlib's spelling is with a z.

Reply:
- fixed in ef31e8d01164fe51bf76146cbad0678a60a9f324

Other places spelling as `normalised`:
- https://github.com/matplotlib/matplotlib/blob/029b026afe65da32f406af618dc3dd209f40120a/lib/matplotlib/axes/_axes.py#L2051
- https://github.com/matplotlib/matplotlib/blob/029b026afe65da32f406af618dc3dd209f40120a/lib/matplotlib/axes/_axes.py#L2131
- https://github.com/matplotlib/matplotlib/blob/029b026afe65da32f406af618dc3dd209f40120a/doc/api/prev_api_changes/api_changes_3.9.0/behaviour.rst#L106